cdap-ui/app/wrangler/components/Wrangler/index.js

```diff
@@ -402,7 +402,7 @@ export default class Wrangler extends Component {
                 </label>
               </div>
 
-              <div className="checkbox">
```

Comment:
Can you remove this change? I have this fixed in https://github.com/caskdata/cdap/pull/7659

It's still missing classes for the label and input as well